Library/Homebrew/utils/analytics.sh

```diff
@@ -1,9 +1,26 @@
 setup-analytics() {
-  [[ -n "$HOMEBREW_NO_ANALYTICS" ]] && return
-
   # User UUID file. Used for Homebrew user counting. Can be deleted and
   # recreated with no adverse effect (beyond our user counts being inflated).
   HOMEBREW_ANALYTICS_USER_UUID_FILE="$HOME/.homebrew_analytics_user_uuid"
+
+  if [[ -n "$HOMEBREW_NO_ANALYTICS" ]]
+  then
+    rm -rf "$HOMEBREW_ANALYTICS_USER_UUID_FILE"
```

Comment:
Was fixed in the merged commit: https://github.com/Homebrew/brew/commit/df489cfde25b9ea59fdf12c67876a18c13dde4aa